mlrun/datastore/azure_blob.py

```diff
@@ -26,37 +27,58 @@
 # Azure blobs will be represented with the following URL: az://<container name>. The storage account is already
 # pointed to by the connection string, so the user is not expected to specify it in any way.
 
+MAX_CONCURRENCY = 100
+MAX_BLOCKSIZE = 1024 * 1024 * 4
+MAX_SINGLE_PUT_SIZE = 1024 * 1024 * 8  # for BlobServiceClient only.
+
 
 class AzureBlobStore(DataStore):
     using_bucket = True
 
     def __init__(self, parent, schema, name, endpoint="", secrets: dict = None):
         super().__init__(parent, name, schema, endpoint, secrets=secrets)
+        self._service_client = None
+        self.max_concurrency = MAX_CONCURRENCY
+        self.max_blocksize = MAX_BLOCKSIZE
+        self.max_single_put_size = MAX_SINGLE_PUT_SIZE
```

Comment:
Why copy these constants to self if you're not going to allow for overriding them with a parameter?